modules/nf-core/unifire/main.nf

```diff
@@ -0,0 +1,57 @@
+process UNIFIRE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    container "dockerhub.ebi.ac.uk/uniprot-public/unifire:2025.1"
+    containerOptions {
+        if (workflow.containerEngine in ['singularity', 'apptainer']) {
+            return "--bind unifire:/volume"
+        } else {
+            return "-v ./unifire:/volume"
+        }
+    }
+
+    input:
+    tuple val(meta), path(faa, stageAs: "unifire/proteins.fasta")
+
+    output:
+    tuple val(meta), path("unifire/predictions_arba.out")         , emit: arba
+    tuple val(meta), path("unifire/predictions_unirule.out")      , emit: unirule
+    tuple val(meta), path("unifire/predictions_unirule-pirsr.out"), emit: pirsr
+    path "versions.yml"                                           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def VERSION = '2025.1'
+    // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+    """
+    # This tool needs a specific folder to be mounted to work.
+    # Run UniFIRE workflow
+    /opt/scripts/bin/unifire-workflow.sh
```

Comment:
Is there no optional argument for this script ?
If so please pass them using `ext.args`

You can find more information on https://nf-co.re/docs/guidelines/components/modules